cmd/eksctl-anywhere/cmd/createcluster.go

```diff
@@ -143,10 +162,30 @@ func (cc *createClusterOptions) createCluster(cmd *cobra.Command, _ []string) er
 		return err
 	}
 
+	if _, err = time.ParseDuration(cc.timeoutOptions.cpWaitTimeout); err != nil {
+		logger.V(0).Info(fmt.Sprintf(timeoutWarningTemplate,
+			cpWaitTimeoutFlag, cc.timeoutOptions.cpWaitTimeout, clustermanager.DefaultControlPlaneWaitStr))
+		cc.timeoutOptions.cpWaitTimeout = clustermanager.DefaultControlPlaneWaitStr
+	}
+
+	if _, err = time.ParseDuration(cc.timeoutOptions.externalEtcdWaitTimeout); err != nil {
+		logger.V(0).Info(fmt.Sprintf(timeoutWarningTemplate,
```

Comment:
Why wouldn't this just return an error if the value is invalid?

Reply:
So the previous env vars functioned this way in that they did not error out. Instead they just defaulted to the default values so I kept the same functionality.